README.md

```diff
@@ -204,14 +204,15 @@ When using `meta`, it will be substituted with `ctrl` (for Windows) **and** `cmd
 
 **Inputs**
 
-| Input         | Type       | Default | Description                                                                                                   |
-| ------------- | ---------- | ------- | ------------------------------------------------------------------------------------------------------------- |
-| selectedItems | Array<any> | /       | Collection of items that are currently selected                                                               |
-| selectOnDrag  | Boolean    | `true`  | Whether items should be selected while dragging                                                               |
-| disabled      | Boolean    | `false` | Disable selection                                                                                             |
-| disableDrag   | Boolean    | `false` | Disable selection by dragging with the mouse. May be useful for mobile.                                       |
-| selectMode    | Boolean    | `false` | If set to `true`, a _toggle_ mode is activated similar to the `toggleSingleItem` shortcut. Useful for mobile. |
-| custom        | Boolean    | `false` | If set to `true`, all default styles for selected items will not be applied.                                  |
+| Input                 | Type       | Default | Description                                                                                                   |
+| --------------------- | ---------- | ------- | ------------------------------------------------------------------------------------------------------------- |
+| selectedItems         | Array<any> | /       | Collection of items that are currently selected                                                               |
+| selectOnDrag          | Boolean    | `true`  | Whether items should be selected while dragging                                                               |
+| disabled              | Boolean    | `false` | Disable selection                                                                                             |
+| disableDrag           | Boolean    | `false` | Disable selection by dragging with the mouse. May be useful for mobile.                                       |
+| selectMode            | Boolean    | `false` | If set to `true`, a _toggle_ mode is activated similar to the `toggleSingleItem` shortcut. Useful for mobile. |
+| custom                | Boolean    | `false` | If set to `true`, all default styles for selected items will not be applied.                                  |
+| toggleWithoutShortcut | Boolean    | `true`  | If set to `false`, items can only be selected when single clicking and applying a keyboard shortcut
```

Comment:
How 'bout we rename this to `selectWithoutShortcut` because it's actually not toggling an item and only prevents the user from selecting an item when this property is set to `false`. Maybe you have another idea for the name. Toggling would be if you set `selectMode` to `true`. This is useful for a mobile view.

Reply:
If I may drop my 2 cents, I would opt for `SelectWithShortcut` (with `false` as default).

Reply:
I was thinking the same whether this should be the other way around. Makes sense I guess. Let's go with it if everyone agrees.